test/sequential/test-crypto-timing-safe-equal-benchmarks.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+
+const crypto = require('crypto');
+
+const BENCHMARK_FUNC_PATH =
+  '../fixtures/crypto-timing-safe-equal-benchmark-func';
```

Comment:
Nit: `common.fixturesDir` but probably no need to worry about that until we see if this even works.